tools/bridge/bridge/confirmation_sender.py

```diff
@@ -142,6 +105,8 @@ def send_confirmation_transactions(self):
             assert transaction is not None
             self.send_confirmation_transaction(transaction)
 
+    run = send_confirmation_transactions
```

Comment:
It seems like we have now two "public" ways to run this: Calling `run` and running all of `self.services`. Can't we just get rid of `run`? As far as I can see, it's only used in tests.

Reply:
I would then rather get rid of send_confirmations_transactions and watch_pending_transactions (i.e. replace them with run) 

Is that fine for you?